deploy/docker/cp-dts-log-checker/dts-log-checker.py

```diff
@@ -0,0 +1,187 @@
+import glob
+import os
+from datetime import datetime
+import re
+import codecs
+import copy
+from pipeline.api import PipelineAPI
+from pipeline.log.logger import LocalLogger, TaskLogger, RunLogger
+
+DTS_LAST_RECORD_FILE = 'last_record.txt'
+DTS_EMAIL_TEMPLATE = 'template.html'
+DTS_LOGS_FILES = 'dts*.*'
```

Comment:
this parameter should be configurable with pipeline run parameters, and `dts*.*` should be used as default